cristin-person/src/main/java/no/unit/nva/cristin/person/model/nva/Person.java

```diff
@@ -61,20 +74,28 @@ public String getContext() {
         return context;
     }
 
+    public void setContext(String context) {
+        this.context = context;
+    }
+
     public List<TypedValue> getIdentifiers() {
-        return Objects.nonNull(identifiers) ? identifiers : Collections.emptyList();
+        return Objects.nonNull(identifiers) ? sortedListOfIdentifiers(identifiers) : Collections.emptyList();
     }
 
     public URI getId() {
         return id;
     }
 
+    public String getType() {
+        return type;
+    }
+
     public void setIdentifiers(List<TypedValue> identifiers) {
         this.identifiers = identifiers;
     }
 
     public List<Affiliation> getAffiliations() {
-        return Objects.nonNull(affiliations) ? affiliations : Collections.emptyList();
+        return Objects.nonNull(affiliations) ? sortedListOfAffiliations(affiliations) : Collections.emptyList();
```

Comment:
sortedListOfAffiliations sorts entries based on hashcode. I imagine this is because some test is failing. I tested the code by replacing the sortedLIstOfAffiliations with affiliations and all tests pass. Therefore I suggest removing the sorting and replacing the lists with sets, because there is no case where we will want to have duplicate TypedValues (i.e. two typed values where both "type" and "value"  are equal)